src/test/regress/Makefile

```diff
@@ -161,16 +161,8 @@ check-follower-cluster: all
 	$(pg_regress_multi_check) --load-extension=citus --follower-cluster \
 	-- $(MULTI_REGRESS_OPTS) --schedule=$(citus_abs_srcdir)/multi_follower_schedule $(EXTRA_TESTS)
 
-COLUMNAR_SCHEDULES =
-COLUMNAR_ISOLATION_SCHEDULES =
-# even though we always add the fdw schedules, keep them separate from the declaration
-# above for easy removabl when fdw support is removed
-COLUMNAR_SCHEDULES += columnar_fdw_schedule
-COLUMNAR_ISOLATION_SCHEDULES += columnar_fdw_isolation_schedule
-ifeq ($(HAS_TABLEAM),yes)
-  COLUMNAR_SCHEDULES += columnar_am_schedule
-  COLUMNAR_ISOLATION_SCHEDULES += columnar_am_isolation_schedule
-endif
+COLUMNAR_SCHEDULES += columnar_am_schedule
+COLUMNAR_ISOLATION_SCHEDULES += columnar_am_isolation_schedule
```

Comment:
If we are not going to check the `HAS_TABLEAM` flag, which I am ok with, we should remove the `check-columnar` and `check-columnar-isolation` tests from the circle ci file. This will make the tests pass. Keeping these tests without the fdw in there is useless imho.

Since there is now no distinction between versions which schedules to run, can we move these schedules inline in `check-columnar` and `check-columnar-isolation`?

For the circle ci file, these are the blocks you want to remove:
 - https://github.com/citusdata/citus/blob/b0ddbbd33aaebeaae92fbb6171b6066f6e3c271a/.circleci/config.yml#L650-L653
 - https://github.com/citusdata/citus/blob/b0ddbbd33aaebeaae92fbb6171b6066f6e3c271a/.circleci/config.yml#L203-L217
 - https://github.com/citusdata/citus/blob/b0ddbbd33aaebeaae92fbb6171b6066f6e3c271a/.circleci/config.yml#L218-L232